src/app/components/MostRead/index.test.tsx

```diff
@@ -221,13 +224,26 @@ describe('MostRead', () => {
 
         pageType: HOME_PAGE,
       },
+      {
+        description:
+          'should not render most read amp when renderCanonicalOnAmp is true',
+        service: 'mundo',
+        mostReadToggle: true,
+        isAmp: true,
+        variant: null,
+        renderCanonicalOnAmp: true,
+        renderExpectation: shouldNotRenderMostReadAmp,
+
+        pageType: MOST_READ_PAGE,
+      },
```

Comment:
Can we add another test with almost the same params, to ensure that the Canonical component is rendered when renderCanonicalOnAmp is true?
```suggestion
      {
        description:
          'should not render most read amp when renderCanonicalOnAmp is true',
        service: 'mundo',
        mostReadToggle: true,
        isAmp: true,
        variant: null,
        renderCanonicalOnAmp: true,
        renderExpectation: shouldNotRenderMostReadAmp,

        pageType: MOST_READ_PAGE,
      },
      {
        description:
          'should render most read canonical when renderCanonicalOnAmp is true',
        service: 'mundo',
        mostReadToggle: true,
        isAmp: true,
        variant: null,
        renderCanonicalOnAmp: true,
        renderExpectation: shouldRenderMostReadCanonical,

        pageType: MOST_READ_PAGE,
      },
```